packages/fxa-auth-server/config/index.ts

```diff
@@ -137,6 +137,15 @@ const conf = convict({
       format: Boolean,
     },
   },
+  googleAuthConfig: {
+    clientId: {
+      default:
+        '218517873053-th4taguk9dvf03rrgk8sigon84oigf5l.apps.googleusercontent.com',
```

Comment:
This is currently pointing to a clientId I generated, we should change this once https://github.com/mozilla/fxa/issues/11314 lands.